store/src/java/com/zimbra/cs/datasource/imap/RemoteFolder.java

```diff
@@ -240,7 +276,7 @@ public void debug(String fmt, Object... args) {
         }
     }
 
-    public void info(String fmt, Object... args) {
+	public void info(String fmt, Object... args) {
```

Comment:
It looks just changing white spaces to a tab. White spaces should be used for indentation as same as others. Revert the change so that it does not appear in the code diff.